src/cdk/schematics/ng-update/migrations/attribute-selectors.ts

```diff
@@ -23,23 +23,23 @@ export class AttributeSelectorsMigration extends Migration<UpgradeData> {
   data = getVersionUpgradeData(this, 'attributeSelectors');
 
   // Only enable the migration rule if there is upgrade data.
-  enabled = this.data.length !== 0;
+  enabled: boolean = this.data.length !== 0;
```

Comment:
I wonder why this is needed. Seems like something TS could infer quite easily?

Reply:
It's a weird one, but for some reason TS was flagging the entire class if the type isn't specified explicitly. It was something about not being able to infer the type from the constructor.